planner-front/src/Components/Home.js

```diff
@@ -23,32 +22,41 @@ export default function Home() {
   const displayHomePage = () => {
     if (!loading) {
       return (
-        <>
+        <body>
```

Comment:
It's not a good idea to use `<body>` tag here. It results in nested `<body>` tags on the page. It happens, because there is already `<body>` tag in `/public/index.html` file.
![double-body](https://user-images.githubusercontent.com/22754636/167686442-3c677761-bb06-449a-8551-4be45c32355b.jpg)

Reply:
You're right. Finally I went back to <> again.